lighthouse-core/config/config.js

```diff
@@ -129,17 +129,13 @@ function validatePasses(passes, audits, rootPath) {
     });
   });
 
-  // Log if multiple passes require trace or network recording and could overwrite one another.
+  // Passes should have unique passName's. Warn otherwise.
```

Comment:
I guess that's hard to differentiate in a comment :)
``// Passes should have unique `passName`s. Warn otherwise.``? 
``// Passes should all have a unique passName. Warn otherwise.``?